src/CMakeLists.txt

```diff
@@ -62,7 +62,13 @@ if (BUILD_TESTING)
   endif()
 
   if (TARGET UNIT_ParamPassing_TEST)
```

Comment:
Looking at the nested ifs and the target_ commands, is this check now superfluous and can be removed?

Reply:
if that is the case we can probably join both instructions related to `UNIT_parser_urdf_TEST` and `UNIT_ParamPassing_TEST` after the same `if (USE_INTERNAL_URDF)` clause.

Reply:
this block is inside `if(BUILD_TESTING)`, but this test does not run on windows, so I think it is important to check that the test target exists. I think there is a more elegant way to express this; let me think about it...

Reply:
ok I've simplified it quite a bit in https://github.com/ignitionrobotics/sdformat/pull/800/commits/2ba37d88796583aa0c4efbb5619eef6b0ccb0774 by using a cmake interface library

Reply:
I believe I've addressed this comment and will merge so I can prepare another prerelease pull request